src/main/java/TestRailIntegration/TestRailIntegration.java

```diff
@@ -0,0 +1,160 @@
+package TestRailIntegration;
+
+import com.codepine.api.testrail.TestRail;
+import com.codepine.api.testrail.model.Project;
+import com.codepine.api.testrail.model.Result;
+import com.codepine.api.testrail.model.Run;
+import com.codepine.api.testrail.model.Suite;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class TestRailIntegration {
+
+    private static TestRail testRail;
+    protected final static String endPoint = "https://kovaloleksandra.testrail.io/";
+    protected final static String username = "oleksandra_koval@epam.com";
+    protected final static String password = "KJv0CdvX0u9arx0jaC2D";
+    protected static int projectId;
```

Comment:
I'm not sure you need to store projectId & suiteId here due to no usage

Reply:
we use those vars in CustomReporter so why they are protected